file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237
        throw new Error(y, { cause: u });
              ^

Error: [23-May-2025 07:28:48 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[23-May-2025 07:29:32 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[23-May-2025 07:30:16 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[23-May-2025 07:32:23 UTC] PHP Deprecated:  Function register_sidebar_widget is <strong>deprecated</strong> since version 2.8.0! Use wp_register_sidebar_widget() instead. in /wordpress/wp-includes/functions.php on line 6121
[23-May-2025 07:32:23 UTC] PHP Deprecated:  Function register_widget_control is <strong>deprecated</strong> since version 2.8.0! Use wp_register_widget_control() instead. in /wordpress/wp-includes/functions.php on line 6121
[23-May-2025 07:32:31 UTC] PHP Deprecated:  Function register_sidebar_widget is <strong>deprecated</strong> since version 2.8.0! Use wp_register_sidebar_widget() instead. in /wordpress/wp-includes/functions.php on line 6121
[23-May-2025 07:32:31 UTC] PHP Deprecated:  Function register_widget_control is <strong>deprecated</strong> since version 2.8.0! Use wp_register_widget_control() instead. in /wordpress/wp-includes/functions.php on line 6121
[23-May-2025 07:32:40 UTC] PHP Deprecated:  Function register_sidebar_widget is <strong>deprecated</strong> since version 2.8.0! Use wp_register_sidebar_widget() instead. in /wordpress/wp-includes/functions.php on line 6121
[23-May-2025 07:32:40 UTC] PHP Deprecated:  Function register_widget_control is <strong>deprecated</strong> since version 2.8.0! Use wp_register_widget_control() instead. in /wordpress/wp-includes/functions.php on line 6121
[23-May-2025 07:32:49 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT error_count from wp_WPMailMon_counters where ID>0</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT `error_count` from `wp_WPMailMon_counters` where `ID` &gt; 0 | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 no such table: wp_WPMailMon_counters
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(683): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 'HY000', Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('SELECT error_co...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('SELECT error_co...')
#3 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SELECT error_co...')
#4 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(687): wpdb->get_var('SELECT error_co...')
#5 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(219): WPMailMon->check_mail_throttle()
#6 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(2653): WPMailMon->__construct()
#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('wpmailmon/wpmai...')
#9 /internal/eval.php(20): activate_plugin('wpmailmon/wpmai...')
#10 {main}</pre>
 for query SELECT error_count from wp_WPMailMon_counters where ID>0 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/wpmailmon/wpmailmon.php'), WPMailMon->__construct, WPMailMon->check_mail_throttle, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:32:49 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT throttle_counter_day from wp_WPMailMon_counters where ID>0</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT `throttle_counter_day` from `wp_WPMailMon_counters` where `ID` &gt; 0 | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 no such table: wp_WPMailMon_counters
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(683): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 'HY000', Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('SELECT throttle...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('SELECT throttle...')
#3 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SELECT throttle...')
#4 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(689): wpdb->get_var('SELECT throttle...')
#5 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(219): WPMailMon->check_mail_throttle()
#6 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(2653): WPMailMon->__construct()
#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('wpmailmon/wpmai...')
#9 /internal/eval.php(20): activate_plugin('wpmailmon/wpmai...')
#10 {main}</pre>
 for query SELECT throttle_counter_day from wp_WPMailMon_counters where ID>0 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/wpmailmon/wpmailmon.php'), WPMailMon->__construct, WPMailMon->check_mail_throttle, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:32:49 UTC] PHP Warning:  Undefined array key "id" in /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-information-schema-builder.php on line 972
[23-May-2025 07:32:49 UTC] PHP Warning:  Trying to access array offset on value of type null in /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-information-schema-builder.php on line 972
[23-May-2025 07:32:49 UTC] PHP Warning:  Undefined array key "id" in /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-information-schema-builder.php on line 981
[23-May-2025 07:32:49 UTC] PHP Warning:  Trying to access array offset on value of type null in /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-information-schema-builder.php on line 981
[23-May-2025 07:32:49 UTC] PHP Warning:  Undefined array key "id" in /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-information-schema-builder.php on line 972
[23-May-2025 07:32:49 UTC] PHP Warning:  Trying to access array offset on value of type null in /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-information-schema-builder.php on line 972
[23-May-2025 07:32:49 UTC] PHP Warning:  Undefined array key "id" in /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-information-schema-builder.php on line 981
[23-May-2025 07:32:49 UTC] PHP Warning:  Trying to access array offset on value of type null in /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-information-schema-builder.php on line 981
[23-May-2025 07:32:49 UTC] PHP Warning:  Undefined array key "id" in /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-information-schema-builder.php on line 972
[23-May-2025 07:32:49 UTC] PHP Warning:  Trying to access array offset on value of type null in /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-information-schema-builder.php on line 972
[23-May-2025 07:32:49 UTC] PHP Warning:  Undefined array key "id" in /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-information-schema-builder.php on line 981
[23-May-2025 07:32:49 UTC] PHP Warning:  Trying to access array offset on value of type null in /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-information-schema-builder.php on line 981
[23-May-2025 07:32:50 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>wpmailmon</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[23-May-2025 07:32:58 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT error_count from wp_WPMailMon_counters where ID>0</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT error_count from wp_WPMailMon_counters where ID&gt;0 | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_WPMailMon_counters.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SELECT error_co...')
#2 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(687): wpdb->get_var('SELECT error_co...')
#3 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(219): WPMailMon->check_mail_throttle()
#4 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(2653): WPMailMon->__construct()
#5 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#6 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('wpmailmon/wpmai...')
#7 /internal/eval.php(20): activate_plugin('wpmailmon/wpmai...')
#8 {main}</pre>
 for query SELECT error_count from wp_WPMailMon_counters where ID>0 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/wpmailmon/wpmailmon.php'), WPMailMon->__construct, WPMailMon->check_mail_throttle, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:32:58 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT throttle_counter_day from wp_WPMailMon_counters where ID>0</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT throttle_counter_day from wp_WPMailMon_counters where ID&gt;0 | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_WPMailMon_counters.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SELECT throttle...')
#2 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(689): wpdb->get_var('SELECT throttle...')
#3 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(219): WPMailMon->check_mail_throttle()
#4 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(2653): WPMailMon->__construct()
#5 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#6 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('wpmailmon/wpmai...')
#7 /internal/eval.php(20): activate_plugin('wpmailmon/wpmai...')
#8 {main}</pre>
 for query SELECT throttle_counter_day from wp_WPMailMon_counters where ID>0 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/wpmailmon/wpmailmon.php'), WPMailMon->__construct, WPMailMon->check_mail_throttle, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:32:58 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE TABLE wp_WPMailMon_throttle_rules (ID INT(9) NOT NULL AUTO_INCREMENT, UNIQUE KEY ID (id), rule_active INT(9) DEFAULT 0, rule_title VARCHAR(200), rule_expires VARCHAR(100), rule_type SET('time','flood') NOT NULL, rule_min INT(9) DEFAULT 0, rule_hour INT(9) DEFAULT 0, rule_day INT(9) DEFAULT 0, time_hour_start VARCHAR(100) DEFAULT NULL, time_hour_end VARCHAR(100) DEFAULT NULL, time_min_start VARCHAR(100) DEFAULT NULL, time_min_end VARCHAR(100) DEFAULT NULL, rule_maxerrors INT(9) DEFAULT 0);</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: MySQL field type cannot be translated to SQLite: set.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(1449): WP_SQLite_DB->query('CREATE TABLE wp...')
#2 /wordpress/wp-includes/class-wp-hook.php(324): WPMailMon->wpmmInstall(false)
#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpmail...', false)
#6 /internal/eval.php(20): activate_plugin('wpmailmon/wpmai...')
#7 {main}</pre>
 for query CREATE TABLE wp_WPMailMon_throttle_rules (ID INT(9) NOT NULL AUTO_INCREMENT, UNIQUE KEY ID (id), rule_active INT(9) DEFAULT 0, rule_title VARCHAR(200), rule_expires VARCHAR(100), rule_type SET('time','flood') NOT NULL, rule_min INT(9) DEFAULT 0, rule_hour INT(9) DEFAULT 0, rule_day INT(9) DEFAULT 0, time_hour_start VARCHAR(100) DEFAULT NULL, time_hour_end VARCHAR(100) DEFAULT NULL, time_min_start VARCHAR(100) DEFAULT NULL, time_min_end VARCHAR(100) DEFAULT NULL, rule_maxerrors INT(9) DEFAULT 0); made by activate_plugin, do_action('activate_wpmailmon/wpmailmon.php'), WP_Hook->do_action, WP_Hook->apply_filters, WPMailMon->wpmmInstall, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:32:58 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, time_hour_start, time_min_start, time_hour_end, time_min_end) VALUES (0, 'All Day Blocked', '0', 'time', '00', '00', '23', '59')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, time_hour_start, time_min_start, time_hour_end, time_min_end) VALUES (0, :param0 , :param1 , :param2 , :param3 , :param4 , :param5 , :param6 ) | parameters: All Day Blocked, 0, time, 00, 00, 23, 59</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_WPMailMon_throttle_rules.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(1456): WP_SQLite_DB->query('INSERT INTO wp_...')
#2 /wordpress/wp-includes/class-wp-hook.php(324): WPMailMon->wpmmInstall(false)
#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpmail...', false)
#6 /internal/eval.php(20): activate_plugin('wpmailmon/wpmai...')
#7 {main}</pre>
 for query INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, time_hour_start, time_min_start, time_hour_end, time_min_end) VALUES (0, 'All Day Blocked', '0', 'time', '00', '00', '23', '59') made by activate_plugin, do_action('activate_wpmailmon/wpmailmon.php'), WP_Hook->do_action, WP_Hook->apply_filters, WPMailMon->wpmmInstall, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:32:58 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, rule_min, rule_hour, rule_day, rule_maxerrors) VALUES (0, 'No Emails After 3 Errors', '0', 'flood', 0, 0, 0, 3)</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, rule_min, rule_hour, rule_day, rule_maxerrors) VALUES (0, :param0 , :param1 , :param2 , 0, 0, 0, 3) | parameters: No Emails After 3 Errors, 0, flood</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_WPMailMon_throttle_rules.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(1466): WP_SQLite_DB->query('INSERT INTO wp_...')
#2 /wordpress/wp-includes/class-wp-hook.php(324): WPMailMon->wpmmInstall(false)
#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpmail...', false)
#6 /internal/eval.php(20): activate_plugin('wpmailmon/wpmai...')
#7 {main}</pre>
 for query INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, rule_min, rule_hour, rule_day, rule_maxerrors) VALUES (0, 'No Emails After 3 Errors', '0', 'flood', 0, 0, 0, 3) made by activate_plugin, do_action('activate_wpmailmon/wpmailmon.php'), WP_Hook->do_action, WP_Hook->apply_filters, WPMailMon->wpmmInstall, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:32:58 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, rule_min, rule_hour, rule_day, rule_maxerrors) VALUES (0, 'Flooding 10 per minute limit.', '0', 'flood', 10, 0, 0, 0)</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, rule_min, rule_hour, rule_day, rule_maxerrors) VALUES (0, :param0 , :param1 , :param2 , 10, 0, 0, 0) | parameters: Flooding 10 per minute limit., 0, flood</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_WPMailMon_throttle_rules.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(1469): WP_SQLite_DB->query('INSERT INTO wp_...')
#2 /wordpress/wp-includes/class-wp-hook.php(324): WPMailMon->wpmmInstall(false)
#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpmail...', false)
#6 /internal/eval.php(20): activate_plugin('wpmailmon/wpmai...')
#7 {main}</pre>
 for query INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, rule_min, rule_hour, rule_day, rule_maxerrors) VALUES (0, 'Flooding 10 per minute limit.', '0', 'flood', 10, 0, 0, 0) made by activate_plugin, do_action('activate_wpmailmon/wpmailmon.php'), WP_Hook->do_action, WP_Hook->apply_filters, WPMailMon->wpmmInstall, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:32:59 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>wpmailmon</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[23-May-2025 07:33:07 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT error_count from wp_WPMailMon_counters where ID>0</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT error_count from wp_WPMailMon_counters where ID&gt;0 | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_WPMailMon_counters.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SELECT error_co...')
#2 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(687): wpdb->get_var('SELECT error_co...')
#3 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(219): WPMailMon->check_mail_throttle()
#4 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(2653): WPMailMon->__construct()
#5 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#6 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('wpmailmon/wpmai...')
#7 /internal/eval.php(20): activate_plugin('wpmailmon/wpmai...')
#8 {main}</pre>
 for query SELECT error_count from wp_WPMailMon_counters where ID>0 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/wpmailmon/wpmailmon.php'), WPMailMon->__construct, WPMailMon->check_mail_throttle, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:33:07 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT throttle_counter_day from wp_WPMailMon_counters where ID>0</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT throttle_counter_day from wp_WPMailMon_counters where ID&gt;0 | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_WPMailMon_counters.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SELECT throttle...')
#2 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(689): wpdb->get_var('SELECT throttle...')
#3 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(219): WPMailMon->check_mail_throttle()
#4 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(2653): WPMailMon->__construct()
#5 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#6 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('wpmailmon/wpmai...')
#7 /internal/eval.php(20): activate_plugin('wpmailmon/wpmai...')
#8 {main}</pre>
 for query SELECT throttle_counter_day from wp_WPMailMon_counters where ID>0 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/wpmailmon/wpmailmon.php'), WPMailMon->__construct, WPMailMon->check_mail_throttle, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:33:07 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE TABLE wp_WPMailMon_throttle_rules (ID INT(9) NOT NULL AUTO_INCREMENT, UNIQUE KEY ID (id), rule_active INT(9) DEFAULT 0, rule_title VARCHAR(200), rule_expires VARCHAR(100), rule_type SET('time','flood') NOT NULL, rule_min INT(9) DEFAULT 0, rule_hour INT(9) DEFAULT 0, rule_day INT(9) DEFAULT 0, time_hour_start VARCHAR(100) DEFAULT NULL, time_hour_end VARCHAR(100) DEFAULT NULL, time_min_start VARCHAR(100) DEFAULT NULL, time_min_end VARCHAR(100) DEFAULT NULL, rule_maxerrors INT(9) DEFAULT 0);</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: MySQL field type cannot be translated to SQLite: set.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(1449): WP_SQLite_DB->query('CREATE TABLE wp...')
#2 /wordpress/wp-includes/class-wp-hook.php(324): WPMailMon->wpmmInstall(false)
#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpmail...', false)
#6 /internal/eval.php(20): activate_plugin('wpmailmon/wpmai...')
#7 {main}</pre>
 for query CREATE TABLE wp_WPMailMon_throttle_rules (ID INT(9) NOT NULL AUTO_INCREMENT, UNIQUE KEY ID (id), rule_active INT(9) DEFAULT 0, rule_title VARCHAR(200), rule_expires VARCHAR(100), rule_type SET('time','flood') NOT NULL, rule_min INT(9) DEFAULT 0, rule_hour INT(9) DEFAULT 0, rule_day INT(9) DEFAULT 0, time_hour_start VARCHAR(100) DEFAULT NULL, time_hour_end VARCHAR(100) DEFAULT NULL, time_min_start VARCHAR(100) DEFAULT NULL, time_min_end VARCHAR(100) DEFAULT NULL, rule_maxerrors INT(9) DEFAULT 0); made by activate_plugin, do_action('activate_wpmailmon/wpmailmon.php'), WP_Hook->do_action, WP_Hook->apply_filters, WPMailMon->wpmmInstall, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:33:07 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, time_hour_start, time_min_start, time_hour_end, time_min_end) VALUES (0, 'All Day Blocked', '0', 'time', '00', '00', '23', '59')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, time_hour_start, time_min_start, time_hour_end, time_min_end) VALUES (0, :param0 , :param1 , :param2 , :param3 , :param4 , :param5 , :param6 ) | parameters: All Day Blocked, 0, time, 00, 00, 23, 59</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_WPMailMon_throttle_rules.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(1456): WP_SQLite_DB->query('INSERT INTO wp_...')
#2 /wordpress/wp-includes/class-wp-hook.php(324): WPMailMon->wpmmInstall(false)
#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpmail...', false)
#6 /internal/eval.php(20): activate_plugin('wpmailmon/wpmai...')
#7 {main}</pre>
 for query INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, time_hour_start, time_min_start, time_hour_end, time_min_end) VALUES (0, 'All Day Blocked', '0', 'time', '00', '00', '23', '59') made by activate_plugin, do_action('activate_wpmailmon/wpmailmon.php'), WP_Hook->do_action, WP_Hook->apply_filters, WPMailMon->wpmmInstall, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:33:07 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, rule_min, rule_hour, rule_day, rule_maxerrors) VALUES (0, 'No Emails After 3 Errors', '0', 'flood', 0, 0, 0, 3)</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, rule_min, rule_hour, rule_day, rule_maxerrors) VALUES (0, :param0 , :param1 , :param2 , 0, 0, 0, 3) | parameters: No Emails After 3 Errors, 0, flood</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_WPMailMon_throttle_rules.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(1466): WP_SQLite_DB->query('INSERT INTO wp_...')
#2 /wordpress/wp-includes/class-wp-hook.php(324): WPMailMon->wpmmInstall(false)
#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpmail...', false)
#6 /internal/eval.php(20): activate_plugin('wpmailmon/wpmai...')
#7 {main}</pre>
 for query INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, rule_min, rule_hour, rule_day, rule_maxerrors) VALUES (0, 'No Emails After 3 Errors', '0', 'flood', 0, 0, 0, 3) made by activate_plugin, do_action('activate_wpmailmon/wpmailmon.php'), WP_Hook->do_action, WP_Hook->apply_filters, WPMailMon->wpmmInstall, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:33:07 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, rule_min, rule_hour, rule_day, rule_maxerrors) VALUES (0, 'Flooding 10 per minute limit.', '0', 'flood', 10, 0, 0, 0)</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, rule_min, rule_hour, rule_day, rule_maxerrors) VALUES (0, :param0 , :param1 , :param2 , 10, 0, 0, 0) | parameters: Flooding 10 per minute limit., 0, flood</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_WPMailMon_throttle_rules.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/wpmailmon/wpmailmon.php(1469): WP_SQLite_DB->query('INSERT INTO wp_...')
#2 /wordpress/wp-includes/class-wp-hook.php(324): WPMailMon->wpmmInstall(false)
#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpmail...', false)
#6 /internal/eval.php(20): activate_plugin('wpmailmon/wpmai...')
#7 {main}</pre>
 for query INSERT INTO wp_WPMailMon_throttle_rules (rule_active, rule_title, rule_expires, rule_type, rule_min, rule_hour, rule_day, rule_maxerrors) VALUES (0, 'Flooding 10 per minute limit.', '0', 'flood', 10, 0, 0, 0) made by activate_plugin, do_action('activate_wpmailmon/wpmailmon.php'), WP_Hook->do_action, WP_Hook->apply_filters, WPMailMon->wpmmInstall, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:33:08 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>wpmailmon</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[23-May-2025 07:33:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT COUNT( * ) AS `Rows` , `userid` FROM `wp_wpstorecart_alerts` GROUP BY `userid` ORDER BY `userid`;</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT COUNT ( * ) AS `Rows` , `userid` FROM `wp_wpstorecart_alerts` GROUP BY `userid` ORDER BY `userid` | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 no such table: wp_wpstorecart_alerts
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(683): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 'HY000', Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('SELECT COUNT( *...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('SELECT COUNT( *...')
#3 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT COUNT( *...')
#4 /wordpress/wp-content/plugins/wpstorecart/wpstorecart/alerts/alerts.php(262): wpdb->get_results('SELECT COUNT( *...', 'ARRAY_A')
#5 /wordpress/wp-content/plugins/wpstorecart/wpstorecart/alerts/alerts.php(592): wpscScanForAlertsAllUsers()
#6 /wordpress/wp-content/plugins/wpstorecart/wpstorecart.php(66): require_once('/wordpress/wp-c...')
#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('wpstorecart/wps...')
#9 /internal/eval.php(20): activate_plugin('wpstorecart/wps...')
#10 {main}</pre>
 for query SELECT COUNT( * ) AS `Rows` , `userid` FROM `wp_wpstorecart_alerts` GROUP BY `userid` ORDER BY `userid`; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/wpstorecart/wpstorecart.php'), require_once('/plugins/wpstorecart/wpstorecart/alerts/alerts.php'), wpscScanForAlertsAllUsers, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:33:35 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT COUNT( * ) AS `Rows` , `userid` FROM `wp_wpstorecart_alerts` GROUP BY `userid` ORDER BY `userid`;</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT COUNT( * ) AS `Rows` , `userid` FROM `wp_wpstorecart_alerts` GROUP BY `userid` ORDER BY `userid`; | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_wpstorecart_alerts.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT COUNT( *...')
#2 /wordpress/wp-content/plugins/wpstorecart/wpstorecart/alerts/alerts.php(262): wpdb->get_results('SELECT COUNT( *...', 'ARRAY_A')
#3 /wordpress/wp-content/plugins/wpstorecart/wpstorecart/alerts/alerts.php(592): wpscScanForAlertsAllUsers()
#4 /wordpress/wp-content/plugins/wpstorecart/wpstorecart.php(66): require_once('/wordpress/wp-c...')
#5 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#6 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('wpstorecart/wps...')
#7 /internal/eval.php(20): activate_plugin('wpstorecart/wps...')
#8 {main}</pre>
 for query SELECT COUNT( * ) AS `Rows` , `userid` FROM `wp_wpstorecart_alerts` GROUP BY `userid` ORDER BY `userid`; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/wpstorecart/wpstorecart.php'), require_once('/plugins/wpstorecart/wpstorecart/alerts/alerts.php'), wpscScanForAlertsAllUsers, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:33:49 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT COUNT( * ) AS `Rows` , `userid` FROM `wp_wpstorecart_alerts` GROUP BY `userid` ORDER BY `userid`;</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT COUNT( * ) AS `Rows` , `userid` FROM `wp_wpstorecart_alerts` GROUP BY `userid` ORDER BY `userid`; | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_wpstorecart_alerts.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT COUNT( *...')
#2 /wordpress/wp-content/plugins/wpstorecart/wpstorecart/alerts/alerts.php(262): wpdb->get_results('SELECT COUNT( *...', 'ARRAY_A')
#3 /wordpress/wp-content/plugins/wpstorecart/wpstorecart/alerts/alerts.php(592): wpscScanForAlertsAllUsers()
#4 /wordpress/wp-content/plugins/wpstorecart/wpstorecart.php(66): require_once('/wordpress/wp-c...')
#5 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#6 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('wpstorecart/wps...')
#7 /internal/eval.php(20): activate_plugin('wpstorecart/wps...')
#8 {main}</pre>
 for query SELECT COUNT( * ) AS `Rows` , `userid` FROM `wp_wpstorecart_alerts` GROUP BY `userid` ORDER BY `userid`; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/wpstorecart/wpstorecart.php'), require_once('/plugins/wpstorecart/wpstorecart/alerts/alerts.php'), wpscScanForAlertsAllUsers, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:34:27 UTC] PHP Deprecated:  Function wpdb::escape is <strong>deprecated</strong> since version 3.6.0! Use wpdb::prepare() or esc_sql() instead. in /wordpress/wp-includes/functions.php on line 6121
[23-May-2025 07:34:36 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE TABLE wp_private_messages (
		id mediumint(9) NOT NULL AUTO_INCREMENT,
		senderid int(11) DEFAULT '0' NOT NULL,
		rcpid int(11) DEFAULT '0' NOT NULL,
		sender VARCHAR(30) NOT NULL,
		recipient VARCHAR(30) NOT NULL,
		subject VARCHAR(255) NOT NULL,
		message text NOT NULL,
		fromsee TINYINT(2),
		tosee TINYINT(2),
		date TIMESTAMP,
		status TINYINT(2),
		ip VARCHAR(15),
	  PRIMARY KEY id (id)
	)</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Cannot combine AUTOINCREMENT and multiple primary keys in SQLite.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE wp...')
#2 /wordpress/wp-content/plugins/wp-private-messages/wpu_private_messages.php(51): dbDelta(Array)
#3 /wordpress/wp-includes/class-wp-hook.php(324): wpu_private_messages_activate(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wp-pri...', false)
#7 /internal/eval.php(20): activate_plugin('wp-private-mess...')
#8 {main}</pre>
 for query CREATE TABLE wp_private_messages (
		id mediumint(9) NOT NULL AUTO_INCREMENT,
		senderid int(11) DEFAULT '0' NOT NULL,
		rcpid int(11) DEFAULT '0' NOT NULL,
		sender VARCHAR(30) NOT NULL,
		recipient VARCHAR(30) NOT NULL,
		subject VARCHAR(255) NOT NULL,
		message text NOT NULL,
		fromsee TINYINT(2),
		tosee TINYINT(2),
		date TIMESTAMP,
		status TINYINT(2),
		ip VARCHAR(15),
	  PRIMARY KEY id (id)
	) made by activate_plugin, do_action('activate_wp-private-messages/wpu_private_messages.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpu_private_messages_activate, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:34:36 UTC] PHP Deprecated:  Function wpdb::escape is <strong>deprecated</strong> since version 3.6.0! Use wpdb::prepare() or esc_sql() instead. in /wordpress/wp-includes/functions.php on line 6121
[23-May-2025 07:34:36 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO wp_private_messages (senderid, rcpid, sender, recipient, subject, message, fromsee, tosee, date, status, ip) VALUES ('0','1','WPU PM','admin','WPU Private Messages','This message mean, you successfully run our plugin! Visit our website for more information!
http://www.wpuzmani.com/
Support Us!', 0, 1, '2025-05-23 07:34:36','0','127.0.0.1')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO wp_private_messages (senderid, rcpid, sender, recipient, subject, message, fromsee, tosee, date, status, ip) VALUES (:param0 ,:param1 ,:param2 ,:param3 ,:param4 ,:param5 , 0, 1, :param6 ,:param7 ,:param8 ) | parameters: 0, 1, WPU PM, admin, WPU Private Messages, This message mean, you successfully run our plugin! Visit our website for more information!
http://www.wpuzmani.com/
Support Us!, 2025-05-23 07:34:36, 0, 127.0.0.1</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_private_messages.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/wp-private-messages/wpu_private_messages.php(66): WP_SQLite_DB->query('INSERT INTO wp_...')
#2 /wordpress/wp-includes/class-wp-hook.php(324): wpu_private_messages_activate(false)
#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wp-pri...', false)
#6 /internal/eval.php(20): activate_plugin('wp-private-mess...')
#7 {main}</pre>
 for query INSERT INTO wp_private_messages (senderid, rcpid, sender, recipient, subject, message, fromsee, tosee, date, status, ip) VALUES ('0','1','WPU PM','admin','WPU Private Messages','This message mean, you successfully run our plugin! Visit our website for more information!
http://www.wpuzmani.com/
Support Us!', 0, 1, '2025-05-23 07:34:36','0','127.0.0.1') made by activate_plugin, do_action('activate_wp-private-messages/wpu_private_messages.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpu_private_messages_activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:34:45 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE TABLE wp_private_messages (
		id mediumint(9) NOT NULL AUTO_INCREMENT,
		senderid int(11) DEFAULT '0' NOT NULL,
		rcpid int(11) DEFAULT '0' NOT NULL,
		sender VARCHAR(30) NOT NULL,
		recipient VARCHAR(30) NOT NULL,
		subject VARCHAR(255) NOT NULL,
		message text NOT NULL,
		fromsee TINYINT(2),
		tosee TINYINT(2),
		date TIMESTAMP,
		status TINYINT(2),
		ip VARCHAR(15),
	  PRIMARY KEY id (id)
	)</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Cannot combine AUTOINCREMENT and multiple primary keys in SQLite.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE wp...')
#2 /wordpress/wp-content/plugins/wp-private-messages/wpu_private_messages.php(51): dbDelta(Array)
#3 /wordpress/wp-includes/class-wp-hook.php(324): wpu_private_messages_activate(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wp-pri...', false)
#7 /internal/eval.php(20): activate_plugin('wp-private-mess...')
#8 {main}</pre>
 for query CREATE TABLE wp_private_messages (
		id mediumint(9) NOT NULL AUTO_INCREMENT,
		senderid int(11) DEFAULT '0' NOT NULL,
		rcpid int(11) DEFAULT '0' NOT NULL,
		sender VARCHAR(30) NOT NULL,
		recipient VARCHAR(30) NOT NULL,
		subject VARCHAR(255) NOT NULL,
		message text NOT NULL,
		fromsee TINYINT(2),
		tosee TINYINT(2),
		date TIMESTAMP,
		status TINYINT(2),
		ip VARCHAR(15),
	  PRIMARY KEY id (id)
	) made by activate_plugin, do_action('activate_wp-private-messages/wpu_private_messages.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpu_private_messages_activate, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:34:45 UTC] PHP Deprecated:  Function wpdb::escape is <strong>deprecated</strong> since version 3.6.0! Use wpdb::prepare() or esc_sql() instead. in /wordpress/wp-includes/functions.php on line 6121
[23-May-2025 07:34:45 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO wp_private_messages (senderid, rcpid, sender, recipient, subject, message, fromsee, tosee, date, status, ip) VALUES ('0','1','WPU PM','admin','WPU Private Messages','This message mean, you successfully run our plugin! Visit our website for more information!
http://www.wpuzmani.com/
Support Us!', 0, 1, '2025-05-23 07:34:45','0','127.0.0.1')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO wp_private_messages (senderid, rcpid, sender, recipient, subject, message, fromsee, tosee, date, status, ip) VALUES (:param0 ,:param1 ,:param2 ,:param3 ,:param4 ,:param5 , 0, 1, :param6 ,:param7 ,:param8 ) | parameters: 0, 1, WPU PM, admin, WPU Private Messages, This message mean, you successfully run our plugin! Visit our website for more information!
http://www.wpuzmani.com/
Support Us!, 2025-05-23 07:34:45, 0, 127.0.0.1</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_private_messages.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/wp-private-messages/wpu_private_messages.php(66): WP_SQLite_DB->query('INSERT INTO wp_...')
#2 /wordpress/wp-includes/class-wp-hook.php(324): wpu_private_messages_activate(false)
#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wp-pri...', false)
#6 /internal/eval.php(20): activate_plugin('wp-private-mess...')
#7 {main}</pre>
 for query INSERT INTO wp_private_messages (senderid, rcpid, sender, recipient, subject, message, fromsee, tosee, date, status, ip) VALUES ('0','1','WPU PM','admin','WPU Private Messages','This message mean, you successfully run our plugin! Visit our website for more information!
http://www.wpuzmani.com/
Support Us!', 0, 1, '2025-05-23 07:34:45','0','127.0.0.1') made by activate_plugin, do_action('activate_wp-private-messages/wpu_private_messages.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpu_private_messages_activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[23-May-2025 07:36:18 UTC] PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function WP_Widget::__construct(), 0 passed in /wordpress/wp-includes/class-wp-widget-factory.php on line 62 and at least 2 expected in /wordpress/wp-includes/class-wp-widget.php:163
Stack trace:
#0 /wordpress/wp-includes/class-wp-widget-factory.php(62): WP_Widget->__construct()
#1 /wordpress/wp-includes/widgets.php(123): WP_Widget_Factory->register('WCZHotPosts')
#2 /wordpress/wp-content/plugins/wcz-hot-posts/wcz-hot-posts.php(78): register_widget('WCZHotPosts')
#3 /wordpress/wp-includes/class-wp-hook.php(324): wcz_hot_posts_widget('')
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-settings.php(727): do_action('init')
#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#10 {main}
  thrown in /wordpress/wp-includes/class-wp-widget.php on line 163
[23-May-2025 07:36:27 UTC] PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function WP_Widget::__construct(), 0 passed in /wordpress/wp-includes/class-wp-widget-factory.php on line 62 and at least 2 expected in /wordpress/wp-includes/class-wp-widget.php:163
Stack trace:
#0 /wordpress/wp-includes/class-wp-widget-factory.php(62): WP_Widget->__construct()
#1 /wordpress/wp-includes/widgets.php(123): WP_Widget_Factory->register('WCZHotPosts')
#2 /wordpress/wp-content/plugins/wcz-hot-posts/wcz-hot-posts.php(78): register_widget('WCZHotPosts')
#3 /wordpress/wp-includes/class-wp-hook.php(324): wcz_hot_posts_widget('')
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-settings.php(727): do_action('init')
#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#10 {main}
  thrown in /wordpress/wp-includes/class-wp-widget.php on line 163
[23-May-2025 07:36:37 UTC] PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function WP_Widget::__construct(), 0 passed in /wordpress/wp-includes/class-wp-widget-factory.php on line 62 and at least 2 expected in /wordpress/wp-includes/class-wp-widget.php:163
Stack trace:
#0 /wordpress/wp-includes/class-wp-widget-factory.php(62): WP_Widget->__construct()
#1 /wordpress/wp-includes/widgets.php(123): WP_Widget_Factory->register('WCZHotPosts')
#2 /wordpress/wp-content/plugins/wcz-hot-posts/wcz-hot-posts.php(78): register_widget('WCZHotPosts')
#3 /wordpress/wp-includes/class-wp-hook.php(324): wcz_hot_posts_widget('')
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-settings.php(727): do_action('init')
#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#10 {main}
  thrown in /wordpress/wp-includes/class-wp-widget.php on line 163

    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
  [cause]: Error: Error when executing the blueprint step #5 ({"step":"installPlugin","pluginData":{"resource":"wordpress.org/plugins","slug":"wp-reviewer"},"options":{"activate":true}}) : Could not download "https://downloads.wordpress.org/plugin/wp-reviewer.latest-stable.zip".
  				Check if the URL is correct and the server is reachable.
  				If it is reachable, the server might be blocking the request.
  				Check the browser console and network tabs for more information.
  
  				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
  
  				This means the server that hosts your file does not allow requests from other sites
  				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
  				cross-origin file downloads. Learn more about CORS at
  				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
  
  				If your file is on GitHub, load it from "raw.githubusercontent.com".
  				Here's how to do that:
  
  				1. Start with the original GitHub URL of the file. For example:
  				https://github.com/username/repository/blob/branch/filename.
  				2. Replace "github.com" with "raw.githubusercontent.com".
  				3. Remove the "/blob/" part of the URL.
  
  				The resulting URL should look like this:
  				https://raw.githubusercontent.com/username/repository/branch/filename
  
  				Error:
  				Error: Could not download "https://downloads.wordpress.org/plugin/wp-reviewer.latest-stable.zip"
      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)
      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
    [cause]: Error: Could not download "https://downloads.wordpress.org/plugin/wp-reviewer.latest-stable.zip".
    				Check if the URL is correct and the server is reachable.
    				If it is reachable, the server might be blocking the request.
    				Check the browser console and network tabs for more information.
    
    				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
    
    				This means the server that hosts your file does not allow requests from other sites
    				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
    				cross-origin file downloads. Learn more about CORS at
    				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
    
    				If your file is on GitHub, load it from "raw.githubusercontent.com".
    				Here's how to do that:
    
    				1. Start with the original GitHub URL of the file. For example:
    				https://github.com/username/repository/blob/branch/filename.
    				2. Replace "github.com" with "raw.githubusercontent.com".
    				3. Remove the "/blob/" part of the URL.
    
    				The resulting URL should look like this:
    				https://raw.githubusercontent.com/username/repository/branch/filename
    
    				Error:
    				Error: Could not download "https://downloads.wordpress.org/plugin/wp-reviewer.latest-stable.zip"
        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)
  }
}

Node.js v23.11.0
